### PR TITLE
D8CORE-4020: teaser views of the publications.

### PR DESCRIPTION
**Pull request by jenbreese**
# NOT READY FOR REVIEW

# Summary
- Adding base styles for SBT

# Review By (Date)
- 3/4

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Clear caches.
3. Add teasers on a page.
4. Verify it looks as design noted: 
https://stanfordits.atlassian.net/browse/D8CORE-4020

# Affected Projects or Products
- Teasers for publications

# Associated Issues and/or People
- D8CORE-4020
- https://github.com/SU-SOE/soe_basic/pull/71

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jenbreese** [comment]
Note: @pookmish the codeclimate error is in webpack and doesn't make sense to me. Do you see the problem?

File: modules/stanford_intranet/lib/scss/base/intranet.scss [hunk 1]
```diff
@@ -1 +1,34 @@
 @charset "UTF-8";
+@import "./stanford_intranet.config";
```
Comment (pookmish) [inline modules/stanford_intranet/lib/scss/base/intranet.scss:2]: is the changes in this file intentional for a PR for the publication teasers??

**Review by pookmish — COMMENTED** [review]
Is this pr for publication teasers or intranet or both? please clarify

File: modules/stanford_intranet/lib/scss/base/intranet.scss (hunk 1 above)
Comment (jenbreese) [reply modules/stanford_intranet/lib/scss/base/intranet.scss:2]: That is unintentional and should not be here. My mistake.

**Comment by jenbreese** [comment]
Branches got muddled. Closing this one.

**Pull request closed without merging**